docs/BR.md

```diff
@@ -994,25 +1026,88 @@ Databases maintained by the CA, its owner, or its affiliated companies do not qu
 
 * CAA checking is optional for certificates for which a Certificate Transparency Precertificate (see [Section 7.1.2.9](#7129-precertificate-profile)) was created and logged in at least two public logs, and for which CAA was checked at time of Precertificate issuance.
 * CAA checking is optional for certificates issued by a Technically Constrained Subordinate CA Certificate as set out in [Section 7.1.2.3](#7123-technically-constrained-non-tls-subordinate-ca-certificate-profile) or [Section 7.1.2.5](#7125-technically-constrained-tls-subordinate-ca-certificate-profile), where the lack of CAA checking is an explicit contractual provision in the contract with the Applicant.
-* For certificates issued prior to July 1, 2021, CAA checking is optional if the CA or an Affiliate of the CA is the DNS Operator (as defined in RFC 7719) of the domain's DNS.
 
 CAs are permitted to treat a record lookup failure as permission to issue if:
 
 * the failure is outside the CA's infrastructure; and
 * the lookup has been retried at least once; and
 * the domain's zone does not have a DNSSEC validation chain to the ICANN root.
 
-CAs MUST document potential issuances that were prevented by a CAA record in sufficient detail to provide feedback to the CAB Forum on the circumstances, and SHOULD dispatch reports of such issuance requests to the contact(s) stipulated in the CAA iodef record(s), if present. CAs are not expected to support URL schemes in the iodef record other than mailto: or https:.
+CAs MUST document potential issuances that were prevented by a CAA record in sufficient detail to provide feedback to the CA/Browser Forum on the circumstances, and SHOULD dispatch reports of such issuance requests to the contact(s) stipulated in the CAA iodef record(s), if present. CAs are not expected to support URL schemes in the iodef record other than mailto: or https:.
+
+#### 3.2.2.9 Multi-Perspective Issuance Corroboration
+
+Multi-Perspective Issuance Corroboration attempts to corroborate the determinations (i.e., domain validation pass/fail, CAA permission/prohibition) made by the Primary Network Perspective from multiple remote Network Perspectives before Certificate issuance. This process can improve protection against equally-specific prefix Border Gateway Protocol (BGP) attacks or hijacks.
+
+Each corroborating Network Perspective's response MUST provide the CA with the necessary information to allow it to affirmatively assess:
+- a. the presence of the expected 1) Random Value, 2) Request Token, 3) IP Address, or 4) Contact Address, as required by the relied upon validation method specified in Sections 3.2.2.4 and 3.2.2.5; and
+- b. the CA's authority to issue to the requested domain(s), as specified in Section 3.2.2.8.
+
```

Comment:
I worry that this can be read as that every response from a network perspective MUST contain both the presense of the Random Value (or likewise for DCV), AND the CAA confirmation. 

I expect the intent here is to state that the CA MUST get a confirming response on both of these items prior to the issuance of certificate, however, I can imagine that CAs may want to do this by using two different calls, thus getting 2 different responses, one for DCV confirmation, one for CAA confirmation. 

```suggestion
Each corroborating Network Perspective's response(s) MUST provide the CA with the necessary information to allow it to affirmatively assess:
- a. the presence of the expected 1) Random Value, 2) Request Token, 3) IP Address, or 4) Contact Address, as required by the relied upon validation method specified in Sections 3.2.2.4 and 3.2.2.5; and
- b. the CA's authority to issue to the requested domain(s), as specified in Section 3.2.2.8.

The CA MAY use either the same set, or different sets of Network Perspectives for Domain Authorization or Control and CAA Record checks
```

Reply:
I'm not fully happy with the suggestion either, if it inspires someone for a better solution, please comment

Reply:
As described in response to your last comment, we think the added flexibility proposed in this update is positive. 

While we have no issues with the proposed language “The CA MAY use either the same set, or different sets of Network Perspectives for Domain Authorization or Control and CAA Record checks.”, we tweaked this a bit as described below. 

Do you feel the following adequately addresses your concern re: implementation flexibility, while also making it clear that both domain validation and CAA MPIC checks are expected? The bold language is modified, the rest of the italics are unchanged. 

_**The CA MAY use either the same set, or different sets of Network Perspectives when performing Multi-Perspective Issuance Corroboration for the required 1) Domain Authorization or Control and 2) CAA Record checks.**_

_**The set of responses from the relied upon Network Perspectives MUST provide the CA with the necessary information to allow it to affirmatively assess:**_

_a. the presence of the expected 1) Random Value, 2) Request Token, 3) IP Address, or 4) Contact Address, as required by the relied upon validation method specified in Sections 3.2.2.4 and 3.2.2.5; and
b. the CA's authority to issue to the requested domain(s), as specified in Section 3.2.2.8._

While making this change, it seems we need to consider the impact of the “and/or" approach to the Quorum Requirements Table. For example, it seems we’ll need to add something like the following bolded text to make it clear that if not using the same set of perspectives for both DV and CAA checks, two quorums are necessary. We will also do a more holistic review of the ballot text to verify whether other changes need to be made. 

_The "Quorum Requirements" Table describes quorum requirements related to Multi-Perspective Issuance Corroboration. **If the CA does NOT rely on the same set of Network Perspectives for both Domain Authorization or Control and CAA Record checks, the quorum requirements MUST be met for both sets of Network Perspectives (i.e.,the Domain Authorization or Control set and the CAA record check set).** Network Perspectives are considered distinct when the straight-line distance between the two States, Provinces, or Countries they reside in is at least 500 km. Network Perspectives are considered "remote" when they are distinct from the Primary Network Perspective and the other Network Perspectives represented in a quorum._

Do you (or others) have any thoughts on this approach?

We pre-staged these changes in https://github.com/ChristopherRC/servercert/commit/07f9c3bac410a8af53ef60755b2788e67f7ab556 in preparation for a subsequent round of discussion.

Reply:
@ChristopherRC  The proposed language above looks great! Thank you